setup.py

```diff
@@ -0,0 +1,23 @@
+import setuptools
+import os
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="pidtree-bcc",
+    version="0.2",
+    author="Matt Carroll",
+    author_email="oholiab@grimmwa.re",
```

Comment:
mattc@yelp.com ?

Reply:
There's more than one commit in this PR, this is the state of the file from an older commit - it's already up to date in the most recent.